### PR TITLE
Pair program - Add Contact Form

### PR DESCRIPTION
**Pull request by CVILLA09**
Description:

This pull request introduces the following changes to the Portfolio project:

Contact Form: A new contact form has been added to the portfolio. This form includes fields for name, email, and message. Each field has specific validation rules as per the project requirements. The form uses the Formspree service to collect the data.

Mobile Version: The portfolio has been made responsive to accommodate mobile devices. The design follows the provided Figma template and uses flexbox for layout. The website is responsive for screen sizes of 375px (mobile) and 768px (desktop).

Code Quality: The code has been checked for linter errors and follows the correct GitHub flow. The work has been documented in a professional way.

Please review these changes and provide your feedback.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -366,3 +366,110 @@ ul {
...
+  background-color: var(--blue);
+  color: white;
+  border-color: white;
+}
+
+.button-form:active {
+  background-color: #2230d2;
+  color: white;
+  border-color: #2230d2;
+}
```
Comment (usorfaitheloho) [inline style.css]: You did well  implementing the project requirement, please implement the following design changes so you'd have a better UI and better user interaction :

-  kindly include the background images for this contact form section to have a really pleasant form suitable for users' aesthetics
- kindly add the placeholder colors implement in the Figma design template
-  please adjust the submit button to be in the same margin as the other input just as it is in the design given
- kindly use the right font-color for the hover property of your button so it looks similar to the design we have 😄 

Your design :
image

Figma design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @CVILLA09  ,
> 
> Great job for implementing some of the Project Requirements, but there are some changes that need to be implemented in order to proceed to the next project 👍
> 
> ## Highlights 👏🏻
> ✔️ Good design implementation ✔️ You used the correct Git Hub-Flow. ✔️ Good README 
> 
> almost 

> ## Required Changes ♻️
> _Check the comments below the review._

> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — APPROVED** [review]
Hi @CVILLA09  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fmalgarini** [comment]
Hi @CVILLA09 👋 , 

Your project is complete! 👏👏  There is nothing else to say other than... it's time to merge it 
🎉 Congratulations! 🎉 


## Highlights
📌 Fixed all isues that you reviewer suggest you to do✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅


## Optional suggestions

[ ] I suggest you to merge this branch  to the main and delete this. Just for stay more organized for the next project. 😀😀 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fmalgarini** [comment]
Hi @CVILLA09 👋 , 

Your project is complete! 👏👏  There is nothing else to say other than... it's time to merge it 
🎉 Congratulations! 🎉 


## Highlights
📌 Fixed all isues that you reviewer suggest you to do✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅



## Optional suggestions

[ ] I suggest you to merge this branch  to the main and delete this. Just for stay more organized for the next project. 😀😀 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**